File: backend/models/tracked_day.go
package models

import (
	"github.com/jinzhu/gorm"
)

type TrackedDay struct {
	gorm.Model

	Uuid       string
	Date       int64
	Day        int64
	Confirmed  bool
	ChoiceId   int64
	ChoiceText string
	UserId     int64
	User       User
}

File: core_backend/core_backend/Models/TrackedDay.cs
using System.ComponentModel.DataAnnotations;

namespace core_backend.Models
{
    public class TrackedDay
    {
        [Key]
        public string Uuid { get; set; }
        public long Date { get; set; }
        public long Day { get; set; }
        public bool Confirmed { get; set; }
        public long ChoiceId { get; set; }
        public string ChoiceText { get; set; }
        public string UserId { get; set; }

    }
}
